src/main/java/nl/tudelft/cse1110/andy/execution/step/RunCodeChecksStep.java

```diff
@@ -13,17 +13,13 @@
 
 public class RunCodeChecksStep implements ExecutionStep {
     @Override
-    public void execute(Context ctx, ResultBuilder result) {
+    public void execute(Context ctx, ResultBuilder result) throws FileNotFoundException {
```

Comment:
I'd prefer to wrap this in a try / catch and throw a RuntimeException. There's nothing the client of this method can do anyways, so, we just let the error bubble up.

Reply:
Sure, I'll change it. The reason I changed it in the first place was that exceptions were getting suppressed in the step itself instead of bubbling up, but a RuntimeException will achieve the same result without the need to change the signature.